src/plugins/intel_cpu/src/nodes/deconv.cpp

```diff
@@ -671,30 +705,42 @@ void Deconvolution::createPrimitive() {
         DnnlMemoryDescPtr inDesc;
         auto wgh_candidate = dnnl::memory::desc(DnnlExtensionUtils::convertToDnnlDims(int8WeightDims), memory::data_type::s8, memory::format_tag::any);
         DnnlMemoryDescPtr outDesc;
+        DnnlMemoryDescPtr biasDesc;
 
         const NodeDesc *selected_pd = getSelectedPrimitiveDescriptor();
         if (selected_pd == nullptr) {
             IE_THROW() << "Preferable primitive descriptor is not set for node " << getName() << ".";
         }
 
         const auto selectedImpl = selected_pd->getImplementationType();
+
         if (isDynamicNode()) {
             std::tie(inDims, outDims) = makeDummyInOutShape();
             initPaddingR(Shape(inDims), Shape(outDims));
-
             auto inDummyDsc = getBaseMemDescAtInputPort(0)->cloneWithNewDims(inDims);
             auto outDummyDsc = getBaseMemDescAtOutputPort(0)->cloneWithNewDims(outDims);
             inDesc = MemoryDescUtils::convertToDnnlMemoryDesc(inDummyDsc);
             outDesc = MemoryDescUtils::convertToDnnlMemoryDesc(outDummyDsc);
+            if (withBiases) {
+                const VectorDims biasVecDims = MemoryDescUtils::makeDummyShape(getInputShapeAtPort(biasPort), 1).getStaticDims();
```

Comment:
Why `makeDummyShape` is used here? `getInputShapeAtPort(biasPort)` should always be static since weights shape is static.

Reply:
Sure, will remove `makeDummyShape`, Thx!